pkg/oc/cli/admin/migrate/migrator.go

```diff
@@ -594,12 +594,14 @@ type migrateTracker struct {
 func (t *migrateTracker) report(prefix string, info *resource.Info, err error) {
 	ns := info.Namespace
 	if len(ns) > 0 {
-		ns = "-n " + ns
+		ns = " -n " + ns
 	}
+	groupResource := info.Mapping.Resource.GroupResource()
+	groupResourceStr := (&groupResource).String()
 	if err != nil {
-		fmt.Fprintf(t.out, "E%s %-10s %s %s/%s: %v\n", timeStampNow(), prefix, ns, info.Mapping.Resource.Resource, info.Name, err)
+		fmt.Fprintf(t.out, "E%s %-10s%s %s/%s: %v\n", timeStampNow(), prefix, ns, groupResourceStr, info.Name, err)
```

Comment:
Is there no feasible way to use our printing stack here? Seem like the whole point of having one was to avoid manually crafting messages like these.

If not feasible to use here, then this is fine. cc @soltysh

Reply:
You are welcome to change it as you see fit, but I want this fixed as-is first.

Reply:
> If not feasible to use here, then this is fine. cc @soltysh

Sort of yes, but we can generalize all of the use-cases. This particular one is doing more than our printing stack does. So I'm fine with this approach. What we could do is to provide some helpers to getting those group resource string, so that this doesn't have to be replicated over and over again, if anything.

Reply:
> What we could do is to provide some helpers to getting those group resource string, so that this doesn't have to be replicated over and over again, if anything.

+1